cronjobs/upload_to_bsz_ftp_server.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+#
+# A tool to upload files to the BSZ FTP server.
+# In earlier times we had a bash script for this which was using the "ftp" command,
+# but due to problems with error handling granularity we switched to python.
+
```

Comment:
We tend not to have a blank line here in Python.

Reply:
done